LEAF_Request_Portal/js/formSearchMultisite.js

```diff
@@ -0,0 +1,1111 @@
+          { width: searchOrigWidth },
+          400,
+          "swing"
+        );
+        $("#" + prefixID + "searchtxt").focus();
+      });
+    });
+    //added for keyboard navigation and accessibility to close advanced search options
+    var searchOrigWidth = 0;
+    $("#" + prefixID + "advancedOptionsClose").on("keydown", function (e) {
+      if (e.keyCode == 13) {
+        localStorage.setItem(localStorageNamespace + ".search", "");
+        $("#" + prefixID + "searchtxt").val("");
+        search("");
+        $("#" + prefixID + "advancedOptionsClose").css("display", "none");
+        $("#" + prefixID + "advancedOptions").slideUp(function () {
+          $("#" + prefixID + "advancedSearchButton").fadeIn();
+          $("#" + prefixID + "searchtxt").css("display", "inline");
+          $("#" + prefixID + "searchtxt").animate(
+            { width: searchOrigWidth },
+            400,
+            "swing"
+          );
+          $("#" + prefixID + "searchtxt").focus();
+        });
+      }
+    });
+
+    $("#" + prefixID + "advancedSearchButton").on("click", function () {
+      searchOrigWidth = $("#" + prefixID + "searchtxt").width();
+      $("#" + prefixID + "advancedSearchButton").fadeOut();
+      $("#" + prefixID + "searchtxt").animate(
+        { width: "0px" },
+        400,
+        "swing",
+        function () {
+          $("#" + prefixID + "searchtxt").css("display", "none");
+          $("#" + prefixID + "advancedOptions").slideDown(function () {
+            $("#" + prefixID + "advancedOptionsClose").fadeIn();
+          });
+          $("#" + prefixID + "advancedOptions").css("display", "inline");
+          chosenOptions();
+          renderPreviousAdvancedSearch();
+          $("#" + prefixID + "widgetMat_0").focus();
+        }
+      );
+    });
+
+    $("#" + prefixID + "advancedSearchApply").on("click", function () {
+      showBusy();
+      generateSearchQuery();
+    });
+    $("#" + prefixID + "addTerm").on("click", function () {
+      newSearchWidget("AND");
+      chosenOptions();
+    });
+    $("#" + prefixID + "orTerm").on("click", function () {
+      newSearchWidget("OR");
+      chosenOptions();
+    });
+
+    $("#" + prefixID + "searchtxt").on("keydown", function (e) {
+      showBusy();
+      timer = 0;
+      if (e.keyCode == 13) {
+        // enter key
+        search($("#" + prefixID + "searchtxt").val());
+      }
+    });
+
+    newSearchWidget();
+  }
+
+  /**
+   * @memberOf LeafFormSearchMultisite
+   */
+  function init() {
+    renderUI();
+
+    intervalID = setInterval(function () {
+      inputLoop();
+    }, 200);
+    if (
+      !/Android|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
+        navigator.userAgent
+      ) &&
+      window.location.search !== ""
+    ) {
+      focus();
+    }
+    if (getLastSearch() == null) {
+      search("*");
+    } else {
+      var lastSearch = getLastSearch();
+
+      var isJSON = true;
```

Comment:
These two variables are declared vars in multiple menthods.  It might be better to declare them outside of the methods once if they absolutely need to be global.